src/vmm/src/rpc_interface.rs

```diff
@@ -334,20 +339,17 @@ impl<'a> PrebootApiController<'a> {
     ///
```

Comment:
could you update the comments : 
"It takes two closures `recv_req` and `respond`"

Reply:
Removed this now redundant comment.